WordPress/gutenberg
# [RNMobile] Tweaks to Badge Component's Styling

`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3642

## Description

This PR includes some tweaks to the badge component's styling. Specifically, it reduces its padding and changes the background color to a darker gray. 

## How has this been tested?

The updated badge component was compared to the original design mocks here:

| Padding | Margin |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/122748139-c5710800-d283-11eb-854a-6f7d21c8e91e.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/122748072-b2f6ce80-d283-11eb-99d2-85d1615d2382.png" width="100%"> |

To view the badge component:

* Navigate to the post editor in Android and add an image block.
* Upload an image to the image block and set it as the post's featured image (via Post Settings).
* Observe the _Featured_ badge to the block's upper right.

## Screenshots 

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/122751342-8e9cf100-d287-11eb-8a23-aa1c7dd66fba.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/122807583-ab571a00-d2c3-11eb-8941-edea6cea07d5.png" width="100%"> |

_Tested on a Pixel 3a emulator, Android 10_

## Types of changes

* A few [tweaks were made to the badge component's _style.css_ file](https://github.com/WordPress/gutenberg/pull/32865/files#diff-59da596260b632571075a233c14c4c1277bdb8937125bc43fa2d153c4e0f79ab), with an updated background colour and margin/padding updates to match the component's initial design mocks.

## Checklist:
